demo/Demo/Menus.elm

```diff
@@ -0,0 +1,222 @@
+module Demo.Menus where
+
+import Dict
+import Html exposing (..)
+import Html.Attributes exposing (..)
+import Effects
+
+import Material.Menu as Menu exposing (..)
+import Material.Grid as Grid
+import Material.Style exposing (Style)
+
+import Demo.Page as Page
+
+
+-- MODEL
+
+
+type alias Index = Int
+
+
+type alias View =
+  Signal.Address Menu.Action -> Menu.Model -> List Style -> List Html -> Html
+
+
+type alias View' =
+  Signal.Address Menu.Action -> Menu.Model -> Html
+
+
+view' : View -> List Style -> Html -> Signal.Address Menu.Action -> Menu.Model -> Html
+view' view coloring elem addr model =
+  view addr model coloring [elem]
+
+
+menus : List (Index, (Style, String))
+menus =
+  [ (Menu.bottomLeft, "Lower left")
```

Comment:
Maybe say in the Html someplace that "lower left" etc. tells you where the menu appears, not where the menu icon is supposed to sit. (I was confused about it when first seeing the demo.)